g2p_programs/__manifest__.py

```diff
@@ -63,6 +63,12 @@
     },
     "demo": [],
     "images": [],
+    "external_dependencies": {
+        "python": [
+            "python-dateutil",
+            "freezegun",
```

Comment:
Freezegun is just needed for running the tests, so it should not be needed here.